src/Widgets/DisplayWidget.vala

```diff
@@ -37,7 +37,7 @@ namespace WingpanelMonitor {
 
         construct {
             valign = Gtk.Align.CENTER;
-            margin_top = 4;
+            margin_top = 0;
```

Comment:
Why remove this? Does the indicators are by default vertically centered now?

Reply:
yes, by default they are vertically centered now. I made a new commit and deleted margin_top. you can see in the screenshot that before changes indicator is showing a bit down and not vertically centered.